Latest Master fails to build (deps not found)

### Prerequisites

Put an `x` into the box that applies:

- [x] This issue describes a bug.
- [ ] This issue describes a feature request.

For more information, see the [Contributing Guidelines](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

### Description

Building from master/source produces the following error:
```
+ make deps
make: *** No rule to make target 'deps'. 
```

`deps` target was removed from `Makefile` in https://github.com/segmentio/terraform-docs/commit/f0a7d374a52927ce889a5c9646f1f40c3eaa8132

--- a/Makefile
+++ b/Makefile
@@ -55,6 +55,9 @@ clean: ## Clean workspace
 	@ $(MAKE) --no-print-directory log-$@
 	rm -rf ./$(BUILD_DIR)
 
+.PHONY: deps
+deps: vendor ## Install dependencies
+
 .PHONY: fmt
 fmt: ## Format all go files
 	@ $(MAKE) --no-print-directory log-$@

